### ISSUE TITLE
What does this error mean?

### ISSUE THREAD

**Issue by Optictrollz** [TheRoboKitten/ZenAii#11]
/home/masternode/Documents/ZenAii-master/commands/sell.js:53
                throw err
                ^

Error: connect ETIMEDOUT 104.20.13.48:443
    at Object._errnoException (util.js:1041:11)
    at _exceptionWithHostPort (util.js:1064:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1153:14)
Starting 2pct markup error sell Of:GNT-BTC In Case of Error

placing sell order at 0.00007674 BTC, 0.00000151 BTC over best ask

not enough asset balance to sell!
Starting Error Sell Of:GNT-BTC In Case of Error

It is correctly placing sell orders. Yet in the console it is saying not enough balance to sell, kinda confused.
also is there a way to reduce getting this:  Poloniex API is down! unable to call getOrder, retrying in 10s

**Comment by Optictrollz** [comment]
ggetBalance error:
{ error: 'Nonce must be greater than 150430136500000. You provided 150430136431400.' }

I get this aswell occasionally in my other terminal, is there a way I can lower the number of calls without reducing order adjust time.

**Comment by TheRoboKitten** [comment]
I normally just let those errors run their course, usually as on complete means a connection issue. Try running on google cloud ubuntu xenial instance (the cheapest one works)

This is one of the reasons I moved my stuff to the cloud :) Let me know if that doesn't work.

And normally, the second sells happen as a back-up.. I set my third sell to 0pct so I can cancel the order twice if I need to on poloniex after hitting the cancel button twice. (like a emergency handbrake kind-of)
